homeassistant/components/litterrobot/update.py

```diff
@@ -43,7 +39,7 @@ async def async_setup_entry(
         for robot in robots
         if isinstance(robot, LitterRobot4)
     ]
-    async_add_entities(entities)
+    async_add_entities(entities, True)
```

Comment:
We also want to define a `SCAN_INTERVAL` constant at the module level that sets a timedelta of the wanted update frequency.